packages/base/package.json

```diff
@@ -60,12 +60,11 @@
     "@naisutech/react-tree": "^3.0.1",
     "@rjsf/core": "^4.2.0",
     "@types/d3-color": "^3.1.0",
-    "@types/three": "^0.134.0",
```

Comment:
Did you mean to remove this? Have you tried updating it to `0.168.0`?

Reply:
No i didn't mean to remove it, when i did `npm install @types/three` it removed by itself and was also not giving any failure.

Reply:
> Have you tried updating it to `0.168.0`?

I'm on it, it adds back the dependency, pushing in a moment.

Reply:
Now it fails xD 🫠

Reply:
Done in [`b1c311a`](https://github.com/jupytercad/JupyterCAD/pull/418/commits/b1c311a9171efd4c8d3c9f0cc4c47b3e469ba138)